nbs/60_medical.imaging.ipynb

```diff
@@ -415,7 +415,7 @@
     {
```

Comment:
<p>All docstrings should be in double quotes: " . Not in tripled double quotes: """</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2772/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='97'/>

Reply:
<p>lol don't know what I was thinking there - all corrected</p>